script/RumpelConfig.sol

```diff
@@ -143,6 +143,16 @@ library RumpelConfig {
     address public constant MAINNET_YT_WSTUSR_26MAR2025 = 0xe0e034AfF49755e80b15594ce3A16d74d1a09b2F;
     address public constant MAINNET_YT_LVLUSD_28MAY2025 = 0x65901Ac9EFA7CdAf1Bdb4dbce4c53B151ae8d014;
     address public constant MAINNET_YT_EBTC_25JUNE2025 = 0x6a162ea0F31dC63Cd154f4fcCDD43B612Df731e9;
+    address public constant MAINNET_YT_SUSDE_30JUL2025 = 0xb7E51D15161C49C823f3951D579DEd61cD27272B;
+    address public constant MAINNET_YT_EUSDE_28MAY2025 = 0x708dD9B344dDc7842f44C7b90492CF0e1E3eb868;
+    address public constant MAINNET_YT_USDE_30JUL2025 = 0x733Ee9Ba88f16023146EbC965b7A1Da18a322464;
+    address public constant MAINNET_YT_LVLUSD_24SEP2025 = 0x946934554a2Bf59039661f971986F0223E906264;
+    address public constant MAINNET_YT_USR_28MAY2025 = 0x77DE4Be22Ecc633416D79371eF8e861Fb1d2cC39;
+    address public constant MAINNET_YT_WSTUSR_24SEP2025 = 0x1E24B022329f3CA0083b12FAF75d19639FAebF6f;
+    address public constant MAINNET_YT_WEETHS_25JUNE2025 = 0xaaC7DB6C2bC926aDE954D69A2d705f059043EA02;
+    address public constant MAINNET_YT_EETH_25JUNE2025 = 0x08AEfe9dFe7818CaaedD94E38e910d2155b7d2b0;
+    address public constant MAINNET_YT_WEETHK_25JUNE2025 = 0x03722CE19e9F5828969D39474a8EfC35c4eA3987;
+    address public constant MAINNET_YT_AGETH_25JUNE2025 = 0xA15a58BCcdDf80A7e9474Cd1abF0c3ad5E823561;
```

Comment:
I couldn't find this address anywhere (I'm seeing `0x0310a860cf7efe8f54ab9b4de49cd071c37fcbcb` for agETH 25 June YTs)

Reply:
ahh looks like this might be the owner address from the audit

Reply:
(unrelated, but curious to see how this one comes through the audit. our vanilla agETH has been a bit of a laggard)

Reply:
updated the address -- yes, lets see. All of the eeth, weETHs, weETHk, and eBTC positions as well.